base/multi.jl

```diff
@@ -424,15 +436,44 @@ function get_bind_addr(w::Worker)
     get(w.config.bind_addr)
 end
 
+"""
+    myid()
+
+Get the id of the current process.
+"""
 myid() = LPROC.id
 
+"""
+    nprocs()
+
+Get the number of available processes.
+"""
 nprocs() = length(PGRP.workers)
+
+"""
+    nworkers()
+
+Get the number of available worker processes. This is one less than `nprocs()`. Equal to
+`nprocs()` if `nprocs() == 1`.
+"""
 function nworkers()
     n = nprocs()
     n == 1 ? 1 : n-1
 end
 
+"""
+    procs()
+
+Returns a list of all process identifiers.
+"""
 procs() = Int[x.id for x in PGRP.workers]
+
+"""
+    procs(pid::Integer)
+
+Returns a list of all process identifiers visible to worker `pid`.
```

Comment:
Returns a list of all process identifiers on the same physical node. Specifically all workers bound to the same ip-address as `pid` are returned.